### PR TITLE
hijack PR #39 and apply review requests

### PR DESCRIPTION
**Pull request by MarcoWorms**
since #39 seems to be almost done but inactive I copied the commit from it, fixed merge conflicts, and applied a requested review

deadlinks seem to be all private repos (I can't access to known haha)

### CODE REVIEW

**Comment by saltyfacu** [comment]
Isn't this file automatically generated from the GitHub repo?

File: docs/developers/v2/DEPLOYMENT.md [hunk 1]
```diff
@@ -129,34 +129,40 @@ vault.setGovernance(ychad.eth)
    ```
 
 1. Monitor `harvest` and `tend` triggers for first few days. Call `harvest`/`tend` manually.
+1. If this is a new vault deployment, test deposit, withdraw, and transfer to ensure functionality is as expected.
```
Comment (saltyfacu) [inline docs/developers/v2/DEPLOYMENT.md:132]: Style thing: should this be 2.?

File: docs/developers/v2/DEPLOYMENT.md [hunk 2]
```diff
@@ -238,7 +244,7 @@ v0.3.5 -> [v0.3.5-1](https://github.com/yearn/yearn-vaults/tree/v0.3.5-1) (compa
 
 1. Before deploying a strategy with [brownie-strategy-mix](https://github.com/yearn/brownie-strategy-mix) make sure your `brownie-config.yml` points to the correct patched vault version, to get a Health Check enabled `BaseStrategy` imported to your strategy.
 
-1. No change should be necessary on your extended `Strategy` logic to interact with the health check contract. **IMPORTANT**: Check your contract size to see if refactoring is needed for compilation.
+1. To interact with the health check contract, no change should be necessary on your extended `Strategy` logic. **IMPORTANT**: Check your contract size to see if refactoring is needed for compilation.
 
 1. Update your unit tests to set to the common Health Check contract [health.ychad.eth](https://etherscan.io/address/0xddcea799ff1699e98edf118e0629a974df7df012)
```
Comment (saltyfacu) [inline docs/developers/v2/DEPLOYMENT.md:249]: Should tho be 2.?

**Review by saltyfacu — CHANGES_REQUESTED** [review]
Please check the numbering of bullets. Some use only 1., and benefits from auto numbering. And other don't. Pick one style

File: docs/developers/v2/DEPLOYMENT.md (hunk 1 above)
Comment (MarcoWorms) [reply docs/developers/v2/DEPLOYMENT.md:132]: markdown automatically increments, I learned it from our docs hahaha https://yearn-docs-git-fork-marcoworms-strategy-checklist.yearn.farm/contributing/documentation/writing-style-guide

Screen Shot 2022-10-14 at 10 48 40 AM

**Review by DarkGhost7 — APPROVED** [review]
lgtm

**Comment by DarkGhost7** [comment]
@MarcoWorms its showing dead links for these
ERROR: 3 dead links found!
[✖] https://github.com/yearn/chief-multisig-officer → Status: 404
[✖] https://docs.yearn.finance/developers/naming-convention → Status: 404
[✖] https://github.com/yearn/yearn-strategies/issues → Status: 404

**Comment by MarcoWorms** [comment]
Nice catch on the naming-convention link, I think both others are private links!

**Pull request merged**